### DISCUSSION TITLE
Add a dialog box for the rotation in the editor

### DISCUSSION THREAD

**Discussion by NovaliX-Dev** [ppy/osu#23269] [Ideas]
The idea : when we click on the rotation circle, a dialog appear, where we can change the rotation of the object.

In this dialog, we will have a text field to type the desired angle and a scrollbar beneath it, for small changes.

What do you think about this ?

**Comment by peppy** [comment]
Yes, there is a plan to add a dialog UI to match osu-stable for those die-hard fans. It will only be accessible via hotkey though, most likely.

If you're looking for the ability to enter a specific angle, I think we can add that to the newer UI for mutating a selection rather than force users to the legacy method of changing it.

**Comment by NovaliX-Dev** [comment]
Yeah i was suggesting just a text input in the new UI style rather than the stable way. 

We can use the one used for the note sound. What do you think ?
